Code/Mantid/docs/source/algorithms/ConvertToReflectometryQ-v1.rst

```diff
@@ -121,6 +129,58 @@ Output:
     Qx Qz
     Ki Kf
     Pz_i + Pz_f Pz_i - Pz_f
+    
+Patch Plot with Dumped Vertexes
+###############################
+
+ConvertToReflectometryQ has the functionality to produce a table of vertexes before they are
+fitted to a normalised polygon. The plotting of these vertexes results in a patch plot that can be
+achieved by running the algorithm below.
+
+Usage
+-----
+**Example - Patch Plot using the Dumped vertexes from ConvertToReflectometryQ**
+
+.. code-block:: python
+
+    import numpy as np
+    import matplotlib
+    from matplotlib.patches import Polygon
+    from matplotlib.collections import PatchCollection
+    import matplotlib.pyplot as plt
+    from matplotlib.colors import LogNorm
+
+    def patch_plot(vertex_table):
+        fig, ax = plt.subplots()
+
+        patches = list()
+        colors = list()
+        polygon_vertexes = list()   
+
+        for vertex in vertex_table:
+            polygon_vertexes.append((vertex['Qx'], vertex['Qy'] ))
+            if len(polygon_vertexes) == 4:
+                poly = Polygon(polygon_vertexes, True,edgecolor='none',linewidth=0)
+                patches.append(poly)
+                colors.append(vertex['CellSignal'])
+                polygon_vertexes = list()
+         
+        p = PatchCollection(patches, cmap=matplotlib.cm.jet,norm=LogNorm(vmin=1e-3, vmax=1e5),linewidths=(0,))
+        p.set_array(np.array(colors))
+        ax.add_collection(p)
+        plt.colorbar(p)
+        axes = plt.gca()
+        axes.set_xlim([-0.0004,0.0004])
+        axes.set_ylim([0,0.2])
+
+        plt.show()
+    
+    threadsafe_call(patch_plot, mtd['vertex_dump'])
```

Comment:
When you add in the processing part of the script (see comment above), pass in the Table created directly, not via the mtd['ws_name'] I used in the example I sent you.